src/whylogs/core/summaryconverters.py

```diff
@@ -118,6 +121,41 @@ def single_quantile_from_sketch(sketch: kll_floats_sketch, quantile: float):
     return type("Object", (), {"quantile": qval[0]})
 
 
+def _calculate_bins(end: float, start: float, n: int, avg_per_bucket: float, max_buckets: int):
+    # Include the max value in the right-most bin
+    end += abs(end) * 1e-7
+    abs_end = abs(end)
+    abs_start = abs(start)
+    max_magnitude = max(abs_end, abs_start)
```

Comment:
I would just name this `magnitude`

Reply:
we could but the value is computed considering both the start and end values, so I wanted to help make it more explicit that this was the largest magnitude have around the bin boundaries.